pkgs/applications/networking/instant-messengers/session-desktop/default.nix

```diff
@@ -33,7 +33,7 @@ stdenvNoCC.mkDerivation {
       name = "Session";
       desktopName = "Session";
       comment = "Onion routing based messenger";
-      exec = "${appimage}/bin/session-desktop-${version}";
+      exec = "session-desktop";
```

Comment:
How about keeping the same structure as before? Something like this?

```suggestion
      exec = "${appimage}/bin/session-desktop";
```

Reply:
I requested this change above

Reply:
Oooh sorry !!! Ignore me then :)